HW1/P2/P2b.py

```diff
@@ -1,3 +1,32 @@
 from P2 import *
+import itertools
+import numpy as np
+import matplotlib.pyplot as plt
+from random import shuffle
 
-# Your code here
+import findspark
+findspark.init()
+from pyspark import SparkContext, SparkConf
+
+conf = SparkConf().setAppName('P2').setMaster('local')
+sc = SparkContext(conf=conf)
+
+# Calculate a mandelbrot set in parallel manner (with load balancing)
+
+data = list(itertools.permutations(range(1, 2001), 2))
```

Comment:
importing spark like that is very nice, i got stuck with copying everything into a terminal window. i'll work to implement this for myself.

also, the "itertools.permutation" is very elegant. as comparison, here is what i did "manually":

+q = range(0,4000)
+X = sc.parallelize(q,100)
+coordinates = X.flatMap(lambda x: [(x,y) for y in q])